Project/Front-End/career-connect/src/components/CompaniesPage.jsx

```diff
@@ -6,6 +6,10 @@ export default function Companies() {
     return (
     <>
         <h1>This is a a Companies Page </h1>
+        <input type="text" placeholder="Company"   />
+    
+  <button disabled={''}onClick={() => {navigate(`/ProfilePage`), window.location.reload(false)}}>Search</button>
```

Comment:
I may be missing something but what is `window.location.reload(false)` doing for you?